app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = list()
+    for person_data in people:
+        person = Person(person_data["name"], person_data["age"])
+    for person in people:
```

Comment:
This loop is identical with the first one so that can be simplified in order to reduce the complexity of the algorithm.

Reply:
> This loop is identical with the first one so that can be simplified in order to reduce the complexity of the algorithm.

not work